src/rest_api/classes/construct/widgets/overview.clj

```diff
@@ -15,35 +16,70 @@
   {:data (:construct/selection-marker construct)
    :description "Coinjection marker for this transgene"})
 
-(defn type-of-construct [construct] ; not finished - need JSON to be fixed for this field
+(defn type-of-construct [construct] ; This does not show up on the page
```

Comment:
@a8wright Good catch! type_of_construct display is fix in the Catalyst template: WormBase/website@d381b5cda4196c955ec57c30ccbbd48592d13d99